frontends/election-manager/src/components/export_election_ballot_package_modal_button.tsx

```diff
@@ -343,7 +343,7 @@ export function ExportElectionBallotPackageModalButton(): JSX.Element {
           <h1>Download Complete</h1>
           <p>
             You may now eject the USB drive. Use the exported ballot package on
-            this USB drive to configure the Ballot Scanner.
+            this USB drive to configure VxScan or VxCentralScan.
```

Comment:
I was guessing here. Is this right? Or should it just say VxCentralScan?

Reply:
yeah this is right